charts/dispatch/charts/application-manager/templates/ingress.yaml

```diff
@@ -0,0 +1,34 @@
+{{- if .Values.ingress.enabled -}}
+{{- $tls := default .Values.global.tls .Values.ingress.tls -}}
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: {{ template "fullname" . }}
+  labels:
+    app: {{ template "name" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+  annotations:
+    kubernetes.io/ingress.class: nginx
+    ingress.kubernetes.io/auth-signin: https://$host/oauth2/start
+    ingress.kubernetes.io/auth-url: http://{{ .Release.Name }}-oauth2-proxy.{{ .Release.Namespace }}.svc.cluster.local/oauth2/auth
+    {{- range $key, $value := .Values.ingress.annotations }}
+      {{ $key }}: {{ $value | quote }}
+    {{- end }}
+spec:
+  rules:
+    - host: {{ default .Values.global.host .Values.ingress.host }}
+      http:
+        paths:
+          - path: {{ .Values.ingress.path }}
+            backend:
+              serviceName: {{ include "fullname" . }}
+              servicePort: {{ .Values.service.externalPort }}
+  {{- if $tls.secretName }}
+  tls:
+    - secretName: {{ $tls.secretName }}
+      hosts:
+        - {{ default .Values.global.host .Values.ingress.host }}
```

Comment:
We/I changed the templates for how the hosts/ingress is defined.  We should be consistent.  Which could mean making all the other charts more "standard" or making this chart like the others.

Reply:
I will double check